dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/persistence/file/batch/BatchFileOrchestratorTest.kt

```diff
@@ -415,6 +438,29 @@ internal class BatchFileOrchestratorTest {
 
     // endregion
 
+    // region getNewWritableFile
+
+    @Test
+    fun `𝕄 return new File 𝕎 getWritableFile() {forceNewFile=true}`() {
+        // Given
+        assumeTrue(fakeRootDir.listFiles().isNullOrEmpty())
+
+        // When
+        val start = System.currentTimeMillis()
+        val result = testedOrchestrator.getWritableFile(forceNewFile = true)
+        val end = System.currentTimeMillis()
+
+        // Then
+        checkNotNull(result)
```

Comment:
I think it is better to use assertion method from AssertJ here instead of the Kotlin stdlib

```suggestion
        assertThat(result).isNotNull()
```

Reply:
hmm yes, I think I just duplicated that function so I need to change it also in the original.

Reply:
Yeah I think this is everywhere in this test class and removing it will imply other changes :(. I would rather keep it like this and maybe open a new PR.